app/services/settings-manager.ts

```diff
@@ -1,16 +1,18 @@
-import { Service, ViewHandler } from 'services/core';
+import { Inject, Service, ViewHandler } from 'services/core';
+import { GreenService } from './green';
 import * as obs from '../../obs-api';
 
-/*
-Eventually this service will be in charge of storing and managing settings profiles
-once the new persistant storage system is finalized. For now it just retrieves settings
-from the backend.
-*/
+// /*
+// Eventually this service will be in charge of storing and managing settings profiles
+// once the new persistant storage system is finalized. For now it just retrieves settings
+// from the backend.
+// */
 
 class SettingsManagerViews extends ViewHandler<{}> {}
 
 export class SettingsManagerService extends Service {
+  @Inject() private greenService: GreenService;
   get videoSettings() {
-    return obs.VideoFactory.legacySettings;
+    return this.greenService.views.videoSettings;
```

Comment:
as far as i can understand, this will reset all of our user's settings to default on application load-- is that not the case?

Reply:
Ah yes. This actually isn't currently being reference anywhere, so I've removed it.